docs/lake/api.md

```diff
@@ -433,11 +433,11 @@ curl -X POST \
 **Example Response**
 
 ```
-{"type":"QueryChannelSet","value":{"channel_id":0}}
+{"type":"QueryChannelSet","value":{"channel":"main"}}
 {"type":{"kind":"record","id":30,"fields":[{"name":"warehouse","type":{"kind":"primitive","name":"string"}},{"name":"count","type":{"kind":"primitive","name":"uint64"}}]},"value":["miami","1"]}
 {"type":{"kind":"ref","id":30},"value":["chicago","2"]}
-{"type":"QueryChannelEnd","value":{"channel_id":0}}
-{"type":"QueryStats","value":{"start_time":{"sec":1658193276,"ns":964207000},"update_time":{"sec":1658193276,"ns":964592000},"bytes_read":55,"bytes_matched":55,"records_read":3,"records_matched":3}}
+{"type":"QueryChannelEnd","value":{"channel":"main"}}
+{"type":"QueryStats","value":{"start_time":{"sec":1723830077,"ns":168637000},"update_time":{"sec":1723830077,"ns":169204000},"bytes_read":55,"bytes_matched":55,"records_read":3,"records_matched":3}}
```

Comment:
Please leave this line as is.

Reply:
@nwt: Hmm. I guess I can. I thought about that but I just chose to take the output wholesale of a re-run of the same API command rather than think it wise to surgically update only the parts I'd _think_ should change and risk not noticing if I'd made the output somehow internally inconsistent in other ways. I agree that in this case that doesn't seem to be the case, so I'll yield to your preference, but I FWIW I feel differently about it.